frontend/packages/console-shared/src/components/dashboard/status-card/HealthItem.tsx

```diff
@@ -39,6 +50,9 @@ const HealthItem: React.FC<HealthItemProps> = React.memo(
                 linkTitle={title}
                 popupTitle={popupTitle}
                 className="co-status-card__popup"
+                position={isHCPresent ? PopoverPosition.bottom : PopoverPosition.top}
+                maxWidth={isHCPresent ? '40rem' : '21rem'}
+                hasAutoWidth={isHCPresent}
```

Comment:
Why this ?

Reply:
Hadas' suggestion from a sync that happened yesterday.

Reply:
```suggestion
                hasAutoWidth={hasAutoWidth}
```